### ISSUE TITLE
Spaces/tabs inconsistency in JavaScript formatter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#17]
```
The typical PhoneGap style is spaces only. However, the default JSDT formatter 
puts in tabs.  (Preferences->JavaScript->Code Style ->Formatter or 
Shift-Control-F) 


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 20 Feb 2011 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=64.  Spaces is now the default format
```

Original comment by `paul.beu...@gmail.com` on 20 Feb 2011 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
